packages/xrpl/test/integration/transactions/payment.test.ts

```diff
@@ -74,7 +74,6 @@ describe('Payment', function () {
   it(
     'Validate Payment transaction API v2: Payment Transaction: Specify Only DeliverMax field',
     async () => {
-      // @ts-expect-error -- DeliverMax is a non-protocol, RPC level field in Payment transactions
```

Comment:
Thanks for removing these declarations. When I wrote the DeliverMax changes, Typescript transpiler complained about them.

Reply:
The changes to `validateRequiredField` and `validateOptionalField` is what resolved them.